play_tester.py

```diff
@@ -9,10 +9,10 @@
 pp = pprint.PrettyPrinter(indent=4)
 
 # play testing section
-print("\n-----Wave 01 test data-----")
-pp.pprint(HORROR_1)
-pp.pprint(FANTASY_1)
-pp.pprint(FANTASY_2)
+# print("\n-----Wave 01 test data-----")
```

Comment:
In the future, if you make changes to a sandbox file like this, you don't need to add it to the staging area/commit it and push up the changes.

It's totally okay that you did so here since this isn't a PR that will get merged into the project, but if you were to submit a PR to actually be merged into a project then someone reviewing your code would probably ask you to remove these changes so they don't get merged into the main code base.